cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -496,6 +491,38 @@ const updateRecords = function (
     .promise();
 };
 
+// getAllAliases gets all aliases out from a string. For example:
+// {"frontend": ["test.foobar.com", "foobar.com"], "api": ["api.foobar.com"]} will return
```

Comment:
I'm trying to remember why does `alias` contain the values for all the environments? Shouldn't we be passing only values set for a particular environment?
`{"frontend": ["test.foobar.com"], "api": ["api.foobar.com"]}` for "test" for example

Reply:
discussed offline. `aliases` contains the value of all the aliases of services deployed in this environment. And the value of string list is reserved for NLB since we might have two aliases for each deployed service then (however the data structure might adjust in future).